CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.9] - 2023-06-21
+
+### Added
+
+- CASM-4368: Implement prototype to delete cray-product-catalog details from main and sub cm for a particular product version
```

Comment:
This should also mention the new retry option being added to the **modify_config_map** function